sql/2023/01/cwvs-by-wordpress-version.sql

```diff
@@ -0,0 +1,108 @@
+# HTTP Archive query to get Core Web Vitals "good" rates by WordPress version.
+#
+# WPP Research, Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# See https://github.com/GoogleChromeLabs/wpp-research/pull/26
+
+CREATE TEMP FUNCTION
+  IS_GOOD(good FLOAT64,
+    needs_improvement FLOAT64,
+    poor FLOAT64)
+  RETURNS BOOL AS ( good / (good + needs_improvement + poor) >= 0.75 );
+CREATE TEMP FUNCTION
+  IS_NON_ZERO(good FLOAT64,
+    needs_improvement FLOAT64,
+    poor FLOAT64)
+  RETURNS BOOL AS ( good + needs_improvement + poor > 0 );
+SELECT
```

Comment:
Despite the GCP BigQuery formatter not doing this, I think it's a good practice to put blank lines between the main query and any separate queries or functions, to improve readability.
```suggestion
CREATE TEMP FUNCTION
  IS_GOOD(good FLOAT64,
    needs_improvement FLOAT64,
    poor FLOAT64)
  RETURNS BOOL AS ( good / (good + needs_improvement + poor) >= 0.75 );

CREATE TEMP FUNCTION
  IS_NON_ZERO(good FLOAT64,
    needs_improvement FLOAT64,
    poor FLOAT64)
  RETURNS BOOL AS ( good + needs_improvement + poor > 0 );

SELECT
```